CHANGELOG.yaml

```diff
@@ -1,3 +1,9 @@
+master:
+  fixed bugs:
+    - >-
+      Fixed an issue in RequestAuth where type was set to previous auth type
```

Comment:
As mentioned in https://github.com/postmanlabs/postman-collection/pull/1101#issuecomment-686995373, why's PR id missing here?